### ISSUE TITLE
klabel with quote in KOOL definition

### ISSUE THREAD

**Issue by cos** [kframework/k-legacy#1927]
```
// TODO(KORE): drop klabel once issues #1913 are fixed
syntax Ids  ::= List{Id,","}           [klabel("'_,_::Ids")]
```

**Comment by daejunpark** [comment]
fixed in fdc04d5976c7e7857cb40aab0f0c1c65d723eb7b

**Comment by cos** [comment]
that was easy -- I thought there was something broken down the pipeline requiring it.
